forest/daemon/src/daemon.rs

```
@@ -306,6 +307,7 @@ pub(super) async fn start(config: Config, detached: bool) -> Db {
     let sync_state = chain_muxer.sync_state_cloned();
     services.spawn(async {
         chain_muxer.await;
+        Ok(())
```

Why ignore the error message here?

---

Fixed